src/components/Button.js

```diff
@@ -11,11 +11,14 @@ import RadiumVars from '../RadiumVariables';
 /*  Stateless Button components, the button content and methods get passed through props now.  Before each
 		button was configured manually.
 */
+/*
+		Added the animation style for the button, passed in through props.
+ */
 const Button = (props) => {
 	return (
 		<button
 			onClick={props.onClick}
-			style={styles.btn__main}
+			style={[styles.btn__main, props.buttonAnimation]}
```

Comment:
This animation style is currently defined in `Cryptotraq` component and passed in from there.